package.json

```diff
@@ -54,6 +56,8 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "jest": "^26.6.3",
-    "react-test-renderer": "^17.0.2"
+    "react-test-renderer": "^17.0.2",
+    "regenerator-runtime": "^0.13.7",
```

Comment:
gonna assume react-test-renderer and regenerator-runtime are needed for these new tests

Reply:
Exactly. Regenerator-runtime gets supertest and babel to play nice together.